### DISCUSSION TITLE
UI/UX improvements

### DISCUSSION THREAD

**Discussion by BrianHanke** [AcademySoftwareFoundation/OpenRV#93] [Ideas]
Howdy all, I wanted to drop by and share some thoughts about Open RV's default UI. I think most would agree that aesthetically it looks dated. There are also a bunch of design choices that make the interface somewhat cluttered and hard to quickly understand. For example, having heading titles in some of the menus instead of flyout submenus. 

I spent a fair amount of time hacking stylesheets, menu scripts and even recompiling with new icons. I was inspired by DaVinci Resolve's Qt style since I find that very clean and easy to use. 

I don't claim to be a UI expert by any means, but I feel like what I cobbled together looks nicer and is somewhat more streamlined. It would be nice to see some attention given to this in the official codebase, upgrading the look and feel of RV to match other great Qt apps like Maya or Resolve. Thanks!

![openrv](https://user-images.githubusercontent.com/59420805/222484846-5fe2dc08-82d8-4cd8-a8ac-99a4d9533cea.png)

**Comment by compaga** [comment]
Hi BrianHanke, this looks pretty nice. We now have an Open RV TSC public meeting, how about you come and do a short presentation/demo of your work so we can discuss it. You can find the meeting here: https://lists.aswf.io/g/open-review-initiative-discussion/topic/97509022

**Comment by rogernelson** [comment]
+1.  It would be interesting to hear about your experience -- how easy/difficult it was, what we can do to make customization easier, and if there's some work you did that can be extracted to facilitate that.

**Comment by BrianHanke** [comment]
@compaga @rogernelson Thanks, I'd be happy to present at the meeting. Will mark April 18 on my calendar. Keep me posted, and if you need to get in touch you can reach me at brianrhanke at outlook.com.

**Comment by rogernelson** [comment]
Thank you! Here is a calendar link to our TSC meeting:
https://lists.aswf.io/g/open-review-initiative-discussion/viewevent?repeatid=50563&eventid=1860395&calstart=2023-04-18

**Reply by guillaume-brossard** [reply]
Hi @BrianHanke. I was wondering if you were planning to attend Siggraph?

**Reply by BrianHanke** [reply]
No, @guillaume-brossard, I'm not attending.
